src/main/java/medicaldoctor/ControllerManager.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package medicaldoctor;
+
+import java.util.HashMap;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Node;
+import javafx.scene.Parent;
+import javafx.scene.layout.StackPane;
+
+
+
+public class ControllerManager extends StackPane{
```

Comment:
seems like you need some autoformatting, there are extra lines, and missing spaces. in netbeans (for me) it is alt+shift+f.